tools/runners/Icarus.py

```diff
@@ -0,0 +1,15 @@
+from BaseRunner import BaseRunner
+
+
+class Icarus(BaseRunner):
+    def __init__(self):
+        super().__init__("icarus")
+
+    def run_cb(self, tmp_dir, params):
+        self.cmd = ['iverilog', '-g2012', '-o iverilog.out']
+
+        for incdir in params['incdirs']:
+            self.cmd.append('-I' + incdir)
+
+        for f in params['files']:
+            self.cmd.append(f)
```

Comment:
Why not just `cmd += params['files']` instead of the loop?

Reply:
This should be now fixed in all the runners